File: src/main/java/io/github/susimsek/springnextjssamples/SpringNextjsSamplesApplication.java
package io.github.susimsek.springnextjssamples;

import io.github.susimsek.springnextjssamples.config.NativeConfig;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ImportRuntimeHints;

@SpringBootApplication
@ImportRuntimeHints(NativeConfig.AppNativeRuntimeHints.class)
public class SpringNextjsSamplesApplication {

    public static void main(String[] args) {
        SpringApplication.run(SpringNextjsSamplesApplication.class, args);
    }

}

File: src/main/java/io/github/susimsek/springnextjssamples/SpringNextjsSamplesApplication.kt
package io.github.susimsek.springnextjssamples

import io.github.susimsek.springnextjssamples.config.NativeConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ImportRuntimeHints

@SpringBootApplication
@ImportRuntimeHints(NativeConfig.AppNativeRuntimeHints::class)
class SpringNextjsSamplesApplication

fun main(args: Array<String>) {
    runApplication<SpringNextjsSamplesApplication>(*args)
}
